src/Microsoft.AspNetCore.Sockets.Http/HttpConnectionDispatcher.cs

```diff
@@ -192,6 +192,11 @@ public HttpConnectionDispatcher(ConnectionManager manager, ILoggerFactory logger
                             Log.PollCanceled(_logger, connection.ConnectionId, existing.TraceIdentifier);
                         }
                     }
+                    else
+                    {
+                        // Hack to indicate that we're handling the first poll.
+                        context.Items.Add("FirstRequest", true);
```

Comment:
I can't comment on whether this is a good idea but `"FirstRequest"` should be a const field somewhere

Reply:
We're changing it to a constructor argument to LongPollingTransport (since it's created on each request), which means no more magic strings.